### PR TITLE
Added Tape item

### PR DESCRIPTION
**Pull request by InziDeveloperMode**

### CODE REVIEW

File: Exiled.API/Features/Items/Tape.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    using InventorySystem.Items.Autosync;
+    using InventorySystem.Items.FlamingoTapePlayer;
+    using PlayerRoles;
+    using Structs;
+    using UnityEngine;
+
+    /// <summary>
+    /// A wrapper class for <see cref="TapeItem"/>.
+    /// </summary>
+    public class Tape : AutosyncItem, IWrapper<TapeItem>
```
[the remark this answers is not in the document]
Comment (InziDeveloperMode) [reply Exiled.API/Features/Items/Tape.cs]: Updated

File: Exiled.API/Features/AudioManager.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    }
+
+    public static void PlayCustomSound(AudioSource audioSource)
+    {
+        if (CustomSound != null && audioSource != null)
+        {
+            audioSource.clip = CustomSound;
+            audioSource.Play();
+        }
+    }
```
Comment (louis1706) [inline Exiled.API/Features/AudioManager.cs:20]: This can't work it's client side only

File: Exiled.API/Features/Items/Tape.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        /// Gets the success sound of the tape.
+        /// </summary>
+        public AudioClip SuccessSound => Base._successSound;
+
+        /// <summary>
+        /// Gets a value indicating whether this item is holsterable.
+        /// </summary>
+        public bool Holsterable => Base.AllowHolster;
+
+        TapeItem IWrapper<TapeItem>.Base => throw new NotImplementedException();
```
Comment (louis1706) [inline Exiled.API/Features/Items/Tape.cs:50]: Just 
TapeItem Base { get; }

File: Exiled.API/Features/Items/Tape.cs [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        /// Gets the Weight of the tape.
+        /// </summary>
+        public float Weight => Base.Weight;
+
+        /// <summary>
+        /// Plays the custom sound associated with this tape using the audio manager.
+        /// </summary>
+        public void PlayCustomSound()
+        {
+            AudioManager.PlayCustomSound(Base.Owner?.GetComponent<AudioSource>());
```
Comment (louis1706) [inline Exiled.API/Features/Items/Tape.cs:62]: This will not working

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Items/Tape.cs [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+        /// <summary>
+        /// Gets the previous owner of this item.
+        /// </summary>
+        public bool Owner => Base.Owner;
+
+        /// <summary>
+        /// Gets the success sound of the tape.
+        /// </summary>
+        public AudioClip SuccessSound => Base._successSound;
```
Comment (louis1706) [inline Exiled.API/Features/Items/Tape.cs:43]: This is client side

File: Exiled.API/Features/Items/Tape.cs [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        /// Gets a value indicating whether this item is holsterable.
+        /// </summary>
+        public bool Holsterable => Base.AllowHolster;
+
+        TapeItem IWrapper<TapeItem>.Base => throw new NotImplementedException();
+
+        /// <summary>
+        /// Gets the Weight of the tape.
+        /// </summary>
+        public float Weight => Base.Weight;
```
Comment (louis1706) [inline Exiled.API/Features/Items/Tape.cs:55]: This is not needed

**Review by louis1706 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**